### PR TITLE
⭐ Added RadioMe

### PR DESCRIPTION
**Pull request by ACertainCoder**
**What's the name of the service?**
RadioMe

**What should the Presence display?**
Informations about the current playing radio station

**Why do you want support for this service?**
This is a new version and simplified of Radio.net to make internet radio easily accessible for everyone.

**Are you able to provide an image for the service? (512x512 minimum)**
logo_big

**What should the presence look like when active?**
Discord_1578168912

### CODE REVIEW

File: RadioMe/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "fr": "Écouter la radio facilement. Stations de radio de toutes les régions d'Allemagne, de France et d'Autriche."
+  },
+  "url": [
+      "radiome.de",
+      "radiome.at",
+      "radiome.fr"
+  ],
+  "version": "1.0",
+  "logo": "https://cdn.discordapp.com/attachments/534018310128402435/660519124732084244/logo.png",
+  "thumbnail": "https://cdn.discordapp.com/attachments/534018310128402435/660518138735099904/thumbnail.png",
```
Comment (Bas950) [inline RadioMe/dist/metadata.json]: Please upload the screenshots somewhere else like imgur, just so they are not on discords cdn

Comment (ACertainCoder) [reply RadioMe/dist/metadata.json]: Done. 👍

File: RadioMe/dist/metadata.json [hunk 2]
```diff
@@ -15,9 +15,9 @@
       "radiome.fr"
   ],
   "version": "1.0",
-  "logo": "https://cdn.discordapp.com/attachments/534018310128402435/660519124732084244/logo.png",
-  "thumbnail": "https://cdn.discordapp.com/attachments/534018310128402435/660518138735099904/thumbnail.png",
+  "logo": "https://imgur.com/a/CGp1Gno",
+  "thumbnail": "https://imgur.com/a/HcPiBTX",
```
Comment (frozaum) [inline RadioMe/dist/metadata.json]: a direct link for those images

Comment (frozaum) [inline RadioMe/dist/metadata.json]: bro you need to directly link the image

Comment (Bas950) [reply RadioMe/dist/metadata.json]: you need the .png/.jpg links (Should also be i.imgur.com then I think)

Comment (ACertainCoder) [reply RadioMe/dist/metadata.json]: I'm sorry, I forgot that part.

**Pull request merged**